parcel-bundler/parcel
# Adds support for PnP

# ↪️ Pull Request

This PR adds support for PnP to the Parcel resolver.

Related: #2635 

## 💻 Examples

Parcel running under [Berry](https://github.com/yarnpkg/berry) now works 💃

## 🚨 Test instructions

Unit tests usually are a bit impractical since it requires to run under a different environment (since it relies on `process.versions.pnp`). Any idea? I tested it locally by following the steps in the getting started (absolutely great experience btw!) and adding a `require('lodash')` in `index.js`.

To reproduce:

- Make a new project with Yarn and cd' into it
- Run `yarn policies set-version berry`
- Run `yarn --version` to confirm you're running with the v2
- Run `yarn add lodash` and add the `require('lodash')` to `index.js`
- Run `yarn parcel index.html`

## ✔️ PR Todo
